packages/xsnap/test/test-boot-lockdown.js

```diff
@@ -186,9 +186,15 @@ test('console - objects should include detail', async t => {
 
   // send all args to print(), which come from console methods
   // filter stack traces so that we're insensitive to line number changes
+  // filter lockdown warnings
   await worker.evaluate(`
-    const skipLineNumbers = s => !s.startsWith('Error: ');
-    send(printed.map(args => args.map(a => a.toString()).filter(skipLineNumbers)))
+  const skip = s =>
+    !s.startsWith('Error: ') && !s.startsWith('Removing intrinsics.');
+  send(
+    printed
+      .map(args => args.map(a => a.toString()).filter(skip))
+      .filter(a => a.length > 0),
+  );
```

Comment:
I did not know about this filter and I don't quite understand it, either before or afterwards. No change suggested. Given its purpose, don't block on my lack of understanding, but I'd appreciate discussing this sometime.

Reply:
This is a regression test for a situation where rich object structure showed up on the console as just `[object Object]`. It works by intercepting `print()`, but lockdown's warnings about removing unrecognized intrinsics also go via `print()`, and the recent XS update has some new stuff such as `String.prototype.at`. It looks like we fixed that particular case (https://github.com/endojs/endo/issues/854 ), but there are others, so I'm filtering them out for the purpose of this test.

Reply:
I refactored this bit to be more clear.